src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -65,16 +69,14 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
         if (cachedParticipations && cachedParticipations.length > 0) {
             this.exerciseService.find(this.exerciseId).subscribe((exerciseResponse: HttpResponse<Exercise>) => {
                 this.exercise = exerciseResponse.body;
-                this.exercise.isAtLeastTutor = this.accountService.isAtLeastTutorInCourse(this.exercise.course);
```

Comment:
why die we remove the isAtLeastTutor property?

Reply:
It is already done in `checkPermission()` in the `ExerciseService`. We additionally have to set the `isAtLeastInstructor` property which is also done in the `checkPermission()` method.